assets/strings/spa.json

```diff
@@ -64,5 +49,19 @@
     "Contact": "Contactar",
     "Documentation": "Documentación",
     "About_Apertium": "Acerca de Apertium",
-    "Apertium_Downloads": "Descargas de Apertium"
+    "Apertium_Downloads": "Descargas de Apertium",
+    "description": "Apertium es una plataforma de traducción automática basada en reglas. Es software libre y se publica bajo los términos de la Licencia Pública General de GNU.",
+    "Downloads_Para": "Las versiones actuales de <b> Apertium toolbox </b>, así como de <b> datos de pares de idiomas </b> están disponibles en <a href = 'https://github.com/apertium' target = '_ blank 'rel =' noopener '> la página de GitHub </a>. Las instrucciones de instalación para Apertium en todas las plataformas principales se proporcionan en la <a href='http://wiki.apertium.org/wiki/Installation' target='_blank' rel='noopener'> Página de instalación de Wiki </a>.",
```

Comment:
The pieces that contain HTML tags have the whitespaces wrong

Reply:
así como *las* de sounds better to me, but @xavivars knows better. "Página de instalación de Wiki" sounds weird to me, needs "la" and maybe "en la Wiki" is better than 'de'.

Reply:
Fully agree with Fran